code/modules/mob/living/basic/space_fauna/netherworld/migo.dm

```diff
@@ -36,10 +36,12 @@
-	if(prob(0.04)) //1 in 20 chance on-load mi-gos will spawn with a miku wig on.
-		base_icon_state = "mi-go-h"
-		name = "hatsune mi-go"
-		desc += " This one is wearing a bright blue wig."
+
+	if(!istype(src, /mob/living/basic/migo/hatsune) && prob(0.1)) // chance on-load mi-gos will spawn with a miku wig on (shiny variant)
+		var/mob/living/basic/migo/hatsune/wow_very_blue = new(get_turf(loc), mapload)
+		qdel(src)
+		return
```

Comment:
```suggestion
		return INITIALIZE_HINT_QDEL
```

Reply:
Waiting for the issue report in the future when someone uses a polymorphic belt on a mi-go, the 1/1000 change rolls, and they delete themself

Reply:
lmfao